### DISCUSSION TITLE
From [v1] to [v2]: StaticMesh, etc.

### DISCUSSION THREAD

**Discussion by 6thVenom** [bepu/bepuphysics2#113] [Q&A] [answered]
Hi, i recently dug an ancient XNA+BEPUv1.2 project and upgraded it to Monogame 3.8+BEPUv2, but the road is hard from v1 to v2.
So, except samples, is there any big v2 documentation anywhere? tutos?

My main issue atm is that i can't use this for static "world" mesh anymore:
```
TriangleMesh.GetVerticesAndIndicesFromModel(collisionModel, out vertices, out indices);
var mesh = new StaticMesh(vertices, indices, new AffineTransform(new Vector3(0, 0, 0)));
space.Add(mesh);
```

As i understood it (looking at how it's done in the samples), you had now to get somehow the (XNA) ModelMesh's triangles list, create a new mesh using it, and add it to the simulation. 

Is that right? how should i go for this please?

**Comment by RossNordby** [comment, marked as answer]
If you haven't seen them already, check out the [getting started documentation](https://github.com/bepu/bepuphysics2/blob/master/Documentation/GettingStarted.md) and [v1 to v2 concept mapping](https://github.com/bepu/bepuphysics2/blob/master/Documentation/UpgradingFromV1.md).

There is no StaticMesh in v2. Instead, you can [create a Mesh shape](https://github.com/bepu/bepuphysics2/blob/master/Demos/DemoMeshHelper.cs#L11) and then [use it with a static](https://github.com/bepu/bepuphysics2/blob/master/Demos/Demos/CompoundTestDemo.cs#L247).

**Reply by 6thVenom** [reply]
Oh thanks, i was not aware about that documentations.
Pretty much what i understood, now i have to found how to get that juicy triangles from XNA/Monogame meshes.
